packages/manager/modules/pci/src/projects/projects.routing.js

```diff
@@ -88,6 +86,48 @@ export default /* @ngInject */ ($stateProvider) => {
         OvhApiCloudProject.v6().delete({ serviceName: project.serviceName })
           .$promise,
 
+      user: /* @ngInject */ (SessionService) => SessionService.getUser(),
+
+      isTrustedZone: /* @ngInject */ (pciFeatures) => {
+        return pciFeatures.isFeatureAvailable(PCI_FEATURES.OTHERS.TRUSTED_ZONE);
```

Comment:
should this feature come from user object? its possible to get if the user is trusted or not?

Reply:
I put it as feature flipp because for the moment it's only available for `trustedNic` and for client that are from `EU` region.
It's a good place i think